src/pages/VoucherRedemption/MoreInfo.tsx

```diff
@@ -43,6 +47,29 @@ const LandingCard = (props: Props) => {
 
 export default LandingCard;
 
+const getColor = (
+  foreground: Boolean,
+  showInfo?: Boolean,
+  inverted?: Boolean
+): string => {
+  const white = 'white';
+  const red = '#ab192e';
+
+  if (!inverted) {
```

Comment:
very minor nit, I think you can reverse the logic here to make the logic cleaner
```
if (inverted) {
 // code
} else {
 // code
}

Reply:
To this and the other comment--so I was trying to spell this out to be as readable as possible, so I kept the negative case on top, as it's the "default" case, and didn't use a second ternary.

That being said, if you think it'd be more readable the other way, than that seems like a sign I should change it!

Reply:
nah totally up you! Just a matter of preference :)